tests/vite-app/favicon.svg

```diff
@@ -0,0 +1,15 @@
+<svg width="410" height="404" viewBox="0 0 410 404" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Can we have this in git? License wise I mean

Reply:
Vite is MIT licensed - but we could also use another favicon like the AWS one